src/pages/academia.js

```diff
@@ -59,39 +125,136 @@ export default () => {
-                            primary
-                            hasArrow={false}
-                        >
-                            Join MongoDB for Academia
-                        </StyledButton>
-                    </HeaderContent>
+                    <Title>MongoDB for Academia</Title>
+
+                    <H3>Teach modern databases with MongoDB</H3>
+
+                    <StyledP>
+                        Enrich your curriculum with MongoDB content and prepare
+                        the developers of tomorrow to build modern applications.
+                    </StyledP>
+
+                    <StyledButton
+                        onClick={() => scrollToRef(formRef)}
+                        primary
+                        hasArrow={false}
+                    >
+                        Join MongoDB for Academia
+                    </StyledButton>
                 </Header>
-            </HeroBanner>
+            </StyledHeroBanner>
 
             <BodyContent>
-                <div ref={formRef}>
-                    <AcademiaSignUpForm />
+                <div>
+                    <OfferingsContent>
+                        <ArticleH2>Teach MongoDB with Confidence</ArticleH2>
+                        <BodyText>
+                            Gain access to MongoDB software and curriculum
+                            content sourced from MongoDB education experts that
+                            easily integrates in your classroom teaching. We can
+                            also consult you with planning a curriculum.
+                        </BodyText>
+                    </OfferingsContent>
+
+                    <OfferingsContent>
+                        <ArticleH2>MongoDB University On-Demand</ArticleH2>
+                        <BodyText>
+                            You and your students will receive special on-demand
+                            access to MongoDB University, with cohort tracking,
+                            usage analytics in addition to great content that
+                            you can incorporate into your curriculum.
+                        </BodyText>
+                    </OfferingsContent>
+
+                    <OfferingsContent>
+                        <ArticleH2>
+                            Connect with Educators Around the World
+                        </ArticleH2>
+                        <BodyText>
+                            Access our MongoDB for Academia community to
+                            collaborate, share tips and get inspired.
+                        </BodyText>
+                    </OfferingsContent>
                 </div>
+                {!isMobile && (
+                    <img src={TeachMongoDBImage} alt="" width="450px" />
+                )}
+            </BodyContent>
+
+            <EligibilitySection>
+                <BodyContent>
+                    <div>
+                        <EligibilityContent>
```

Comment:
Some stakeholders might want smaller padding around here on mobile, but I think it is worth holding off on messing with it until later.